Source/WebCore/Modules/WebGPU/GPUPresentationContext.cpp

```diff
@@ -41,10 +41,39 @@ void GPUPresentationContext::unconfigure()
     m_backing->unconfigure();
 }
 
-GPUTexture* GPUPresentationContext::getCurrentTexture()
+void GPUPresentationContext::clearCurrentTextureAndView()
 {
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=250958 Figure out how the lifetime of these objects should behave.
-    return nullptr;
+    m_currentTexture = nullptr;
+    m_currentTextureView = nullptr;
+}
+
+void GPUPresentationContext::ensureCurrentTextureAndView()
+{
+    ASSERT(static_cast<bool>(m_currentTexture) == static_cast<bool>(m_currentTextureView));
```

Comment:
could also be written as: `ASSERT(!!m_currentTexture == !!m_currentTextureView);` 

!! is pretty standard for conversion to bool throughout webkit and much shorter.

Reply:
Ugh I hate `!!` :( It sacrifices clarity for brevity

Reply:
I'm going to intentionally not abide by this review comment 🤔

Reply:
Oh, conveniently, this code goes away when I update for https://github.com/WebKit/WebKit/pull/9263#issuecomment-1407257039

Reply:
👍 

Also totally fine to ignore the !! convention if it comes up in the future, just a suggestion :)